src/perl/Permabit/VolumeGroup.pm

```diff
@@ -457,13 +488,18 @@ sub getFreeBytes {
 sub getLVMConfig {
   my ($self, $config) = assertMinMaxArgs([""], 1, 2, @_);
```

Comment:
This function might be misnamed now since it can return a --devices argument as well. But sine it's local to this file it doesn't matter that much, I suppose.
 
I'm also not sure what the doc text about an "additional option' is referring to; is that the --devices argument?

Reply:
See above